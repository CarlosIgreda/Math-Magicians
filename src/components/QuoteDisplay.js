import React from 'react';
import PropTypes from 'prop-types';

const QuoteDisplay = ({ quote }) => (
  <div>
    <p className="quote-text">
      &quot;
      {quote}
      &quot;
    </p>
  </div>
);

QuoteDisplay.propTypes = {
  quote: PropTypes.string.isRequired,
};

export default QuoteDisplay;
