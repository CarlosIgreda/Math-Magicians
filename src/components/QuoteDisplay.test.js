import React from 'react';
import { render } from '@testing-library/react';
import QuoteDisplay from './QuoteDisplay';

describe('QuoteDisplay', () => {
  it('should render the quote correctly', () => {
    const quote = 'Test Quote';
    const { container } = render(<QuoteDisplay quote={quote} />);
    expect(container).toMatchSnapshot();
  });
});
