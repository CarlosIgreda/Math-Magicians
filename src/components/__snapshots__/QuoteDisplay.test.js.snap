// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuoteDisplay should render the quote correctly 1`] = `
<div>
  <div>
    <p
      class="quote-text"
    >
      "
      Test Quote
      "
    </p>
  </div>
</div>
`;
